pkg/types/aws/platform.go

```diff
@@ -129,6 +135,50 @@ type Platform struct {
 	UserProvisionedDNS dns.UserProvisionedDNS `json:"userProvisionedDNS,omitempty"`
 }
 
+// EIPAllocations contains Elastic IP (EIP) allocations for AWS resources
+// within the cluster.
+type EIPAllocations struct {
+	// ingressNetworkLoadBalancer is a list of IDs for Elastic IP (EIP) addresses that
```

Comment:
gofmt
```suggestion
	// IngressNetworkLoadBalancer is a list of IDs for Elastic IP (EIP) addresses that
```

Reply:
from my understanding this shall be small case as we need to specify the field name. even gofmt did not change it capital.

Reply:
@JoelSpeed  ^^

Reply:
I stand corrected!

Reply:
Yes, generally for APIs we prefer to document the fields using the serialised version, so that when the docs get generated for end users, they see the same formatting that they would use in the configuration